example.config.yml

```diff
@@ -15,6 +15,30 @@ vagrant_ip: 192.168.88.88
 # See: https://docs.vagrantup.com/v2/networking/public_network.html
 vagrant_public_ip: ""
 
+# If you want to use a Amazon Web Services Server
+# This will remain off unless aws_key_id != ''
+# If you turn this setting on, you must make a 
+# .pem file that points to the path you specify
+# containing the private key of the Amazon Instance.
+# aws_key_id: Retrieved from the users section of AWS
+# aws_secret_access_key: Also retrieved from users sections of AWS
+# aws_key_pair_name: The name of the KeyPair you generated on AWS
+#                    MUST BE EXACT spelling (Caps, Etc).
+# aws_security_group: The group name you want the security to use.
+# aws_tags_name: What you want the AWS tag name to show up as.
+# override_ssh_username: The SSH username. Should be ubuntun 90% of the time.
+# override_ssh_private_key_path: The path to the private key for the keypair
+#                                you created.
+# The ssh username is default to ubuntu
+aws_key_id: ""
+aws_secret_access_key: ""
+aws_key_pair_name: "KeyPairName_CaSeSeNsItIvE"
```

Comment:
Hmm, good point.
I guess because the user renames the file from example.config.yml to config.yml we could add that to the gitignore?